shared_model/interfaces/query_responses/impl/transactions_page_response.cpp

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "interfaces/query_responses/transactions_page_response.hpp"
+#include "interfaces/transaction.hpp"
+
+namespace shared_model {
+  namespace interface {
+
+    std::string TransactionsPageResponse::toString() const {
+      return detail::PrettyStringBuilder()
+          .init("TransactionsPageResponse")
+          .appendAll(transactions(), [](auto &tx) { return tx.toString(); })
```

Comment:
you could use
```suggestion
          .appendAll("transactions", transactions(), [](auto &tx) { return tx.toString(); })
```
which is defined in https://github.com/hyperledger/iroha/blob/c04712dd6aeb38dfefb4b20c68b0d765bbf04052/shared_model/utils/string_builder.hpp#L73-L75